cypress/component/rich-text/RichTextEditor.Tables.spec.ts

```diff
@@ -168,7 +168,7 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
         BLOCKS.HEADING_4,
         BLOCKS.HEADING_5,
         BLOCKS.HEADING_6,
-      ].forEach((type) => getDropdownItem(type).get('button').should('not.be.disabled'));
```

Comment:
@colomolo I removed the `.get('button')` here from this test, from what I saw the `getDropdownItem` already selects the button, or did I miss something?